recipes/ntp/ntp.inc

```diff
@@ -77,7 +77,7 @@ DEFAULT_USE_ntpd_sysvinit_stop	= "25"
 
 inherit auto-package-utils
 AUTO_PACKAGE_UTILS = "ntpd ntpdate ntp-wait ntp-keygen sntp ntpq ntpdc \
-    ntptime ntptrace tickadj"
+    ntptime ntpsweep ntptrace tickadj calc_tickadj update-leap"
```

Comment:
Looking through the autoconf magic, it looks like it should get installed by default, but can be disabled by setting NTPSWEEP_NI environment variable when calling ./configure